src/Api/Translation.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+namespace FAPI\PhraseApp\Api;
+
+use FAPI\PhraseApp\Model\Translation\TranslationDeleted;
+use FAPI\PhraseApp\Model\Translation\Translation as TranslationModel;
+use GuzzleHttp\Psr7\Response;
+use Psr\Http\Message\ResponseInterface;
+use FAPI\PhraseApp\Exception;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
```

Comment:
You should add your name. The names here is the developers "who knows most about this class"